diff --git a/node_modules/eslint-plugin-codegen/dist/presets/barrel.js b/node_modules/eslint-plugin-codegen/dist/presets/barrel.js
index 713ce15..5f2dff7 100644
--- a/node_modules/eslint-plugin-codegen/dist/presets/barrel.js
+++ b/node_modules/eslint-plugin-codegen/dist/presets/barrel.js
@@ -61,8 +61,7 @@ const barrel = ({ meta, options: opts }) => {
         .sync(pattern, { cwd, ignore: opts.exclude })
         .filter(f => path.resolve(cwd, f) !== path.resolve(meta.filename))
         .map(f => `./${f}`.replace(/(\.\/)+\./g, '.'))
-        .filter(file => ['.js', '.mjs', '.ts', '.tsx'].includes(path.extname(file)))
-        .map(f => f.replace(/\.\w+$/, ''));
+        .map(f => ['.js', '.mjs', '.ts', '.tsx'].includes(path.extname(f)) ? f.replace(/\.\w+$/, '') : f);
     const expectedContent = io_ts_extra_1.match(opts.import)
         .case(undefined, () => {
         return relativeFiles.map(f => `export * from '${f}'`).join('\n');
